plugins/autostart/Cargo.toml

```diff
@@ -13,6 +13,13 @@ links = "tauri-plugin-autostart"
 rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.platforms.support]
+windows = { level = "full", notes = "" }
+linux = { level = "full", notes = "" }
+macos = { level = "full", notes = "" }
```

Comment:
Add that you may not be able to push to the Appstore with this plugin.
See [node-auto-launch](https://github.com/Teamwork/node-auto-launch?tab=readme-ov-file#mac). Should this be in here or the readme?

Reply:
> Should this be in here or the readme?

Both, maybe linking to https://github.com/Teamwork/node-auto-launch/issues/43 in the readme too. (All the way at the bottom are electron PRs that kinda show how to implement it in a store compatible way but that probably has to be done in tauri and not in the plugin but we'll see when we get there)